VAMobile/src/api/types/PersonalInformationData.ts

```diff
@@ -0,0 +1,24 @@
+export type PersonalInformationPayload = {
+  data: {
+    id: string
+    type: string
+    attributes: {
+      firstName: string
+      middleName: string | null
+      lastName: string
+      signinEmail: string
+      signinService: string
+      birthDate: string | null
+    }
+  }
+}
+
+export type PersonalInformationData = {
+  firstName: string
+  middleName: string | null
+  lastName: string
+  signinEmail: string
+  signinService: string
+  birthDate: string | null
+  fullName: string
+}
```

Comment:
```suggestion
export type PersonalInformationData = PersonalInformationPayload['data']['attributes'] & {
  fullName: string
}
```

Another way this can be written